src/Rules/Honeypot.php

```diff
@@ -87,32 +87,18 @@ public static function precheck(): void {
 		if ( strpos( $request_path, 'wp-comments-post.php' ) === false ) {
 			return;
 		}
-		$fields = [];
-		foreach ( $_POST as $key => $value ) {
-			if ( isset( $fields['plugin_field'] ) ) {
-				$fields['hidden_field'] = $key;
-				break;
-			}
-			if ( HoneypotField::get_secret_name_for_post() === $key ) {
-				$fields['plugin_field'] = $key;
-			}
-		}
 
-		if ( ! isset( $fields['plugin_field'] ) ) {
-			// Honeypot field was not present in $_POST data.
-			$_POST['ab_spam__invalid_request'] = 1;
```

Comment:
There is a test case for this field that now fails:
https://github.com/pluginkollektiv/antispam-bee/blob/78e84de0d1eee3a9b2a670cf04921fdd5ba4fad7/tests/Unit/Rules/HoneypotTest.php#L72-L74

Should be either adjusted or removed if obsolete.

Reply:
This test was removed. I've also changed the field names in the tests, since they are currently not dynamic in the way proposed in the tests. The hidden field is always `comment` for this comment form. This is currently hardcoded in our plugin (as it was in v2).